examples/jsm/loaders/SVGLoader.js

```diff
@@ -3082,7 +3082,8 @@ class SVGLoader extends Loader {
 						} else {
 
 							tempV2_3.toArray( vertices, 1 * 3 );
-							tempV2_3.toArray( vertices, 3 * 3 );
+							// using tempV2_4 to update 3rd vertex if the uv.y of 3rd vertex is 1
```

Comment:
The PR indeed fixes the issue and I don't see any regressions in `webgl_loader_svg`.

Do you mind explaining the reasoning behind this change though? Why does the code have to make a distinction at this line?

Reply:
`tempV2_3` and `tempV2_4` are the two new vertices after the cap is calculated. They should replace the corresponding old vertices. Updates to  `tempV2_3` and `tempV2_4` are explicit. But there are two cases of updating vertices[3] with `tempV2_3` or `tempV2_4`.

<img width="400" alt="image" src="https://github.com/mrdoob/three.js/assets/17698194/01857c90-3c90-4641-8cf3-7bd8407894c5">

`uv.y` is used to determine whether to use `tempV2_3` or `tempV2_4`.
<img width="272" alt="image" src="https://github.com/mrdoob/three.js/assets/17698194/5c90fee6-f9f6-4aeb-96be-2615f2f1ffd9">

Reply:
It would be better if there was no need to make a distinction. Maybe there's a better way?

Reply:
Not sure. If somebody comes up with a uniform approach, we can sill update the code. For now, let's go with the this PR.